specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2022-12-01-preview/elasticsan.json

```diff
@@ -0,0 +1,2390 @@
+        "operation",
+        "description"
+      ],
+      "description": "Metadata about an operation.",
+      "additionalProperties": false,
+      "properties": {
+        "provider": {
+          "type": "string",
+          "description": "Localized friendly form of the resource provider name."
+        },
+        "resource": {
+          "type": "string",
+          "description": "Localized friendly form of the resource type related to this action/operation."
+        },
+        "operation": {
+          "type": "string",
+          "description": "Localized friendly name for the operation, as it should be shown to the user."
+        },
+        "description": {
+          "type": "string",
+          "description": "Localized friendly description for the operation, as it should be shown to the user."
+        }
+      }
+    },
+    "ElasticSanRPOperation": {
+      "type": "object",
+      "required": [
+        "name"
+      ],
+      "description": "Description of a ElasticSan RP Operation",
+      "additionalProperties": false,
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the operation being performed on this particular object"
+        },
+        "isDataAction": {
+          "type": "boolean",
+          "description": "Indicates whether the operation applies to data-plane.",
+          "readOnly": true
+        },
+        "display": {
+          "description": "Additional metadata about RP operation.",
+          "$ref": "#/definitions/ElasticSanOperationDisplay",
+          "readOnly": true
+        }
+      }
+    },
+    "ElasticSanOperationListResult": {
+      "type": "object",
+      "required": [
+        "value"
+      ],
+      "description": "List of operations supported by the RP.",
+      "additionalProperties": false,
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "An array of operations supported by the ElasticSan RP.",
+          "items": {
+            "$ref": "#/definitions/ElasticSanRPOperation"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "description": "URI to fetch the next section of the paginated response.",
+          "type": "string"
+        }
+      }
+    },
+    "ElasticSanList": {
+      "type": "object",
+      "description": "List of Elastic Sans",
+      "additionalProperties": false,
+      "properties": {
+        "value": {
+          "description": "An array of Elastic San objects.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ElasticSan"
+          }
+        },
+        "nextLink": {
+          "description": "URI to fetch the next section of the paginated response.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ElasticSanCreateParameter": {
+      "type": "object",
+      "description": "Input Parameter for ElasticSan request.",
```

Comment:
The description looks still not right.
This is only used for create ElasticSan, but not all ElasticSan request. 
Please correct it.
How about "Input Parameter for creating ElasticSan."

Reply:
This is the PUT payload, so it is for both create and update.